omod/src/main/webapp/admin/maintenance/implementationIdForm.jsp

```diff
@@ -30,7 +30,7 @@
 				<td style="white-space: nowrap">
 					<input type="text" value="${status.value}" name="${status.expression}" maxlength="20" size="8"/>
```

Comment:
do we expect cross site scripting in such cases,ie on the input tag

Reply:
If the input field is loaded in with data from the server initially. So if a script is stored on the server, it could be executed when loaded. That was my thought process, but it could be incorrect. What do you think @isears?

Reply:
Are the patches in `implementationIdForm.jsp` still related to EMPT54? So far I've only been able to trigger XSS in the error messages that come up in `headerFull.jsp` but maybe I'm missing something.